EasyVCR/EasyVCR.csproj

```diff
@@ -0,0 +1,65 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <Nullable>enable</Nullable>
+        <TargetFrameworks>netstandard2.0;netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+        <LangVersion>8</LangVersion>
+        <RootNamespace>EasyVCR</RootNamespace>
+        <OutputType>Library</OutputType>
+        <RootNamespace>EasyVCR</RootNamespace>
+        <AssemblyName>EasyVCR</AssemblyName>
+        <Configurations>Release;Debug</Configurations>
+        <Platforms>AnyCPU</Platforms>
+        <PackageId>EasyVCR</PackageId>
+        <LangVersion>8.0</LangVersion>
+        <ImplicitUsings>disable</ImplicitUsings>
+    </PropertyGroup>
+
+    <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+        <DebugSymbols>true</DebugSymbols>
+        <DebugType>full</DebugType>
+        <Optimize>false</Optimize>
+        <OutputPath>bin\Debug</OutputPath>
+        <DefineConstants>TRACE;DEBUG;NET45</DefineConstants>
+        <ErrorReport>prompt</ErrorReport>
+        <WarningLevel>4</WarningLevel>
+        <Prefer32Bit>false</Prefer32Bit>
+    </PropertyGroup>
+    
+    <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'">
+        <DebugType>pdbonly</DebugType>
+        <Optimize>true</Optimize>
+        <OutputPath>..\lib\net</OutputPath>
+        <DefineConstants>TRACE</DefineConstants>
+        <ErrorReport>prompt</ErrorReport>
+        <WarningLevel>4</WarningLevel>
+        <DocumentationFile>..\lib\net\EasyVCR.XML</DocumentationFile>
+        <NoWarn>1701;1702;1591;1570</NoWarn>
+    </PropertyGroup>
+    
+    <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+        <SignAssembly>false</SignAssembly>
+    </PropertyGroup>
+    
+    <!--
+    <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+        <SignAssembly>true</SignAssembly>
+        <AssemblyOriginatorKeyFile>..\EasyPost.pfx</AssemblyOriginatorKeyFile>
+    </PropertyGroup>
+    -->
```

Comment:
We either need to use this or remove it. We shouldn't leave commented out blocks of code.

Since this is to sign it, we should definitely have it present here.

Reply:
Will need to troubleshoot, was getting an error that .NET Core 3.1 could not sign.